content/rancher/v2.x/en/installation/ha/_index.md

```diff
@@ -2,6 +2,78 @@
+
+When Rancher is deployed in HA on a k8s/rke cluster, access to it is provided by ingress controller, a l7 load balancer. Ingress controller uses its own certificates by default. 
+
+Additionally, external lb's could be used to provide access to Rancher. External lb's uses its own certificates by default.
+
+Depending where tls termination is configured, 3 `tls-modes` could be used:
+
+- `ingress`: tls termination is configured at k8s ingress. Connection between ingress and Rancher is proxied in clear. 
+  - Set FQDN dns round robin A entry pointing to RKE nodes ip's.
+  - K8s ingress will get configured with tls and FQDN site.
+  - Ingress requests to FQDN will be forwarded to rancher service in plain.
+
+<img src="./rancher-HA-SSL_ingress.jpg" width="800" alt="Rancher HA tls ingress">
+
+- `external`: tls termination is configured at external lb. Connection between external lb to ingress and finally Rancher is proxied in clear.
+  - Configure external LB with tls (ALB, nginx, haproxy) and FQDN site. L7 proxy.
+  - Set FQDN dns A entry pointing to external LB ip.
+  - External lb's request to FQDN will be forwarded to k8s ingress in plain.
+  - Ingress requests will be forwarded to rancher service in plain.
+  - CA cert is required if ca is selfsigned.
+
+<img src="./rancher-HA-SSL_external.jpg" width="800" alt="Rancher HA tls external">
+
+- `rancher`: Rancher service as tls termination. Connection is ciphered to Rancher.
+  - Set FQDN dns round robin A entry pointing to RKE nodes ip's.
+  - K8s ingress will get configured with FQDN site.
+  - Ingress requests to FQDN will be forwarded to rancher service in https.
+
+<img src="./rancher-HA-SSL_rancher.jpg" width="800" alt="Rancher HA tls rancher">
+
+Rancher tls certs could be managed in distinct ways. Tls could be provided manually at deploy time, or cert-manager integration could be done with 2 dsitinct issuers.
+
+Depending how tls certs are managed, 3 `tls-issuers` could be used:
+
+- `<blank>`: Tls certs will be managed manually.
+  - Tls key and certificate should be provided at deploy time.
+  - CA cert is required if ca is selfsigned.
+
+- `ca`: Tls certs will be managed by cert-manager with ca issuer.
+  - CA key and cert should be provided at deploy time.
+  - Tls certs will be generated and signed automatically by provided ca. 
+
+- `acme`: Tls certs will be managed by cert-manager with acme issuer.
+  - acme-email is required.
+  - Tls certs will be generated and signed automatically by letsencrypt. 
+
+
+**Deployment modes**
+
+- RancherOnRKE: Rancher Kubernetes Engine (RKE) is a fast, versatile k8s installer that you can use to install k8s cluster on your Linux nodes. RKE cluster would be used exclusively to run rancher.
```

Comment:
Add hyperlink to the corresponding pages

Reply:
Ok